.changeset/heavy-toys-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/icons': major
+---
+
+Added a new asset to the icons package. The 'minus_small' icon.
```

Comment:
This will be added to the changelog for `@sumup/icons`, so the context is implied.

```suggestion
Added a new minus icon in a small size.
```